app/views/katello/api/v2/host_collections/_host_collection.json.rabl

```diff
@@ -1,4 +1,5 @@
 attribute :pulp_id, :name, :organization_id, :max_hosts, :description, :total_hosts, :unlimited_hosts
+attributes :host_ids
```

Comment:
@jlsherrill, @waldenraines, updated the behavior to move the host_ids from index to show/create/copy.  In addition, renamed the rabl to use the 'base' convention to be consistent w/ other apis, such as products and repositories.